### PR TITLE
Fix some outdated docs regarding output and checkpoint backends

### PR DESCRIPTION
**Pull request by sbastrakov**
This was noticed during a mattermost discussion on checkpoints.

### CODE REVIEW

**Review by psychocoderHPC — DISMISSED** [review]

**Comment by sbastrakov** [comment]
Sorry, found out a missing `not`, fixed and rebased.

File: docs/source/usage/plugins/checkpoint.rst [hunk 1]
```diff
@@ -62,7 +62,6 @@ PIConGPU command line option                  Description
 Depending on the available external dependencies (see above), the options for the ``<IO-backend>`` are:
```
Comment (steindev) [inline docs/source/usage/plugins/checkpoint.rst:62]: Would you be so kind and remove the double back ticks in line 54 as well?

**Review by steindev — CHANGES_REQUESTED** [review]
Nice catch! Few more spots to be corrected though.

And since you are editing the `TBG_macros.cfg` section of the documentation anyways: I suggest to add a comment to this cfg file, saying that this is really just a collection of snippets and not a working cfg file.
There is the
```bash
"$TBG_cfgPath"/submitAction.sh
```
missing at the end.
I actually stumbled upon this a few days ago, but @psychocoderHPC said it is just supposed to be a collection of snippets.
Although I think we might want to turn it into a full cfg by adding this line, since the file has the `.cfg` ending anyways and thus should be fully functional from my point of view.

**Comment by sbastrakov** [comment]
Thanks @steindev , I've pushed another commit implementing your suggestion and one more thing I've noticed along the way.

**Comment by psychocoderHPC** [comment]
> Nice catch! Few more spots to be corrected though.
> 
> And since you are editing the `TBG_macros.cfg` section of the documentation anyways: I suggest to add a comment to this cfg file, saying that this is really just a collection of snippets and not a working cfg file.
> There is the
> 
> ```shell
> "$TBG_cfgPath"/submitAction.sh
> ```
> 
> missing at the end.
> I actually stumbled upon this a few days ago, but @psychocoderHPC said it is just supposed to be a collection of snippets.
> Although I think we might want to turn it into a full cfg by adding this line, since the file has the `.cfg` ending anyways and thus should be fully functional from my point of view.

We can add it but IMO the file `TBG_macros.cfg` should be deleted and in the docu for each plugin there should a complete example with all options for each plugin that can be copy-pasted by the user.
The reason is that this will provide one point for the user to find all necessary information.

I am fine if you merge this PR first.

**Comment by steindev** [comment]
@sbastrakov added the note to the `cfg`. Which, I think, is sufficient. I still like the idea to have this one place as an overview and example and get additional information via `picongpu --help`.

**Review by steindev — APPROVED** [review]

**Pull request merged**